ballerina/Ballerina.toml

```diff
@@ -1,7 +1,7 @@
 [package]
 org = "ballerina"
 name = "grpc"
-version = "1.4.0"
+version = "1.4.1"
```

Comment:
Shall we move to next minor release?

Reply:
Updated in https://github.com/ballerina-platform/module-ballerina-grpc/pull/1015/commits/16a3b833a595b5b618f6b7e516a361cefcb430a9